widget/gridlayout.go

```diff
@@ -137,45 +137,39 @@ func (g *GridLayout) Layout(widgets []PreferredSizeLocateableWidget, rect image.
 	firstStretchedCol, firstStretchedRow := true, true
 	for _, w := range widgets {
 		cw := colWidths[c]
-		if w.GetWidget().Visibility == Visibility_Hide {
-			c++
-			continue
-		} else if w.GetWidget().Visibility == Visibility_Hide {
-			c++
-			x += cw + g.columnSpacing
-			continue
-		}
-
-		if g.columnStretched(c) {
-			cw = stretchedColWidth
-			if firstStretchedCol {
-				cw = firstStretchedColWidth
-				firstStretchedCol = false
+		ch := rowHeights[r]
+		if w.GetWidget().Visibility == Visibility_Show {
```

Comment:
I basically inverted the 2 ifs with `Visibility_Hide` and `Visibility_Hide_Blocking` and wrapped the code outside both of them with this if insted.